### PR TITLE
Add projects

### PR DESCRIPTION
**Pull request by SaladRaider**
### What does this PR do?
Adds the projects model and api controller.

### Status
READY

CC: @michellejkwon

### CODE REVIEW

File: test/fixtures/projects.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (michellejkwon) [inline test/fixtures/projects.yml]: was fixtures auto-generated? i don't think we'll really need them if we're not going to use them for testing, so you could remove this file.

but if you meant to do this, could you rename the fixtures so it's more representative of the projects?

File: test/models/project_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (michellejkwon) [inline test/models/project_test.rb]: this one too ^

File: test/fixtures/projects.yml (hunk 1 above)
Comment (SaladRaider) [reply test/fixtures/projects.yml]: yeah these were autogenerated. i'll get rid of them

**Comment by SaladRaider** [comment]
@michellejkwon made the changes

**Pull request merged**